kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image-ssl/create-domain-inputs.yaml

```diff
@@ -0,0 +1,165 @@
+#
+# A unique value is required for each domain that is created using the scripts.
+#
+# If you are running the sample scripts from a machine that is remote to the Kubernetes cluster where
+# the domain is going to be running, you need to set this property to the image name that is intended
+# to be used in a registry local to that Kubernetes cluster. You also need to push the `image` to
+# that registry before starting the domain using the `kubectl create -f` or `kubectl apply -f` command.
+# See README.md for more help.
+#image:
+
+# Image pull policy
+# Legal values are "IfNotPresent", "Always", or "Never"
+imagePullPolicy: IfNotPresent
+
+# Name of the Kubernetes secret to access the Docker Store to pull the WebLogic Server Docker image
+# The presence of the secret will be validated when this parameter is enabled.
+#imagePullSecretName:
+
+# Name of the Kubernetes secret for the Admin Server's username and password
+# The name must be lowercase.
+# If not specified, the value is derived from the domainUID as <domainUID>-weblogic-credentials
+weblogicCredentialsSecretName: domain1-weblogic-credentials
+
+# Whether to include server .out to the pod's stdout.
+# The default is true.
+includeServerOutInPodLog: true
+
+# Specifies whether the log home is stored on the persistent volume.
+# If set to true, then you must specify the 'logHome', 'persistentVolumeClaimName'
+# and 'domainPVMountPath' properties.
+# The default is false.
+logHomeOnPV: false
+
+# The in-pod location for domain log, server logs, server out, and node manager log files
+# If not specified, the value is derived from the domainUID as /shared/logs/<domainUID>
+# This parameter is required if 'logHomeOnPV' is true.
+# Otherwise, it is ignored.
+logHome: /shared/logs/domain1
+
+# An (optional) in-pod location for data storage of default and custom file stores.
+# If not specified or the value is either not set or empty (e.g. dataHome: "") then the
+# data storage directories are determined from the WebLogic domain home configuration.
+#dataHome:
+
+# Port for the T3Channel of the NetworkAccessPoint
+t3ChannelPort: 30012
+
+# Public address for T3Channel of the NetworkAccessPoint.  This value should be set to the
+# kubernetes server address, which you can get by running "kubectl cluster-info".  If this
+# value is not set to that address, WLST will not be able to connect from outside the
+# kubernetes cluster.
+#t3PublicAddress: 
+
+# Boolean to indicate if the channel should be exposed as a service
+exposeAdminT3Channel: false
+
+# NodePort to expose for the admin server
+adminNodePort: 30701
+
+# Boolean to indicate if the adminNodePort will be exposed
+exposeAdminNodePort: false
+
+# Name of the domain namespace
+namespace: default
+
+# Java Option for WebLogic Server
+# -Dweblogic.security.SSL.ignoreHostnameVerification=true is necessary in order for the managed server(s) to connect to the admin server
+# while booting up if SSL is enabled.
```

Comment:
Was there going to be some sample related documentation on the HNV setting?

Either way would suggest little more clarification of HNV for demo certs would not match k8s deployment topology

Maybe just something simple like "... while booting up if SSL is enabled since the demo certificate hostname would not match the Kubernetes deployment topology"